exercises/en/exc_03_04.py

```diff
@@ -0,0 +1,14 @@
+import altair as alt
+import pandas as pd
+
+
+penguins = pd.read_csv('data/penguins.csv').dropna(subset=['sex'])
+
+culmen_stacked_plot = ___.___.encode(
+    alt.X(___),
+    alt.Y(___),
+    color=___
+    ).properties(___, width=230, height=200
+    )
```

Comment:
Can we add a nice label for the x axis instead of the column name default?

Reply:
Added `title='Culmen depth (mm)'`